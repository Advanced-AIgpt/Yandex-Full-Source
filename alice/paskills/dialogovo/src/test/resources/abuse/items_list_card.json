{
  "response": {
    "text": "блять ты че совсем охуел",
    "tts": "блять ты че совсем охуел",
    "end_session": false,
    "card": {
      "type": "ItemsList",
      "header": {
        "text": "блять ты че совсем охуел"
      },
      "items": [
        {
          "image_id": "1652229/16b371e1747fa842018d",
          "title": "блять ты че совсем охуел",
          "description": "блять ты че совсем охуел",
          "button": {
            "text": "блять ты че совсем охуел",
            "url": "http://example.com/",
            "payload": {
              "text": "блять ты че совсем охуел"
            }
          }
        },
        {
          "image_id": "1652229/16b371e1747fa842018d",
          "title": "блять ты че совсем охуел",
          "description": "блять ты че совсем охуел",
          "button": {
            "text": "блять ты че совсем охуел",
            "url": "http://example.com/",
            "payload": {}
          }
        }
      ],
      "footer": {
        "text": "блять ты че совсем охуел",
        "button": {
          "text": "блять ты че совсем охуел",
          "url": "https://example.com/",
          "payload": {}
        }
      }
    }
  },
  "version": "1.0"
}
