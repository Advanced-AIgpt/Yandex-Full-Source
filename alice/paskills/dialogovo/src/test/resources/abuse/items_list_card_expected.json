{
  "response": {
    "text": "<censored> ты че совсем <censored>",
    "tts": "<speaker audio=\"beep_mat_1.opus\"> ты че совсем <speaker audio=\"beep_mat_1.opus\">",
    "end_session": false,
    "card": {
      "type": "ItemsList",
      "header": {
        "text": "<censored> ты че совсем <censored>"
      },
      "items": [
        {
          "image_id": "1652229/16b371e1747fa842018d",
          "title": "<censored> ты че совсем <censored>",
          "description": "<censored> ты че совсем <censored>",
          "button": {
            "text": "<censored> ты че совсем <censored>",
            "url": "http://example.com/",
            "payload": {
              "text": "блять ты че совсем охуел"
            }
          }
        },
        {
          "image_id": "1652229/16b371e1747fa842018d",
          "title": "<censored> ты че совсем <censored>",
          "description": "<censored> ты че совсем <censored>",
          "button": {
            "text": "<censored> ты че совсем <censored>",
            "url": "http://example.com/",
            "payload": {}
          }
        }
      ],
      "footer": {
        "text": "<censored> ты че совсем <censored>",
        "button": {
          "text": "<censored> ты че совсем <censored>",
          "url": "https://example.com/",
          "payload": {}
        }
      }
    }
  },
  "version": "1.0"
}
